src/main/resources/kibana-rest.yml

```
@@ -180,6 +191,7 @@ kibana_status:
 kibana_synthetics_monitor_filters:
   versions:
     ">= 8.12.0": "/internal/synthetics/monitor/filters"
+    # ">= 9.0.0": "uri [/internal/synthetics/monitor/filters] with method [get] exists but is not available with the current configuration"
```

Does this mean we should add `< 9.0.0` to the above line to disable it?

```suggestion
    ">= 8.12.0 < 9.0.0": "/internal/synthetics/monitor/filters"
    # ">= 9.0.0": "uri [/internal/synthetics/monitor/filters] with method [get] exists but is not available with the current configuration"
```